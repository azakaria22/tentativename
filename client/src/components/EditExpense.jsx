import React, { Component } from 'react';

class EditExpense extends Component {
  constructor(props) {
    super(props);
    this.state = {
      expense: 'Buffalo Wild Wings',
      cost: 20,
      category: 'Food',
      frequency: 'Once',
      date: '10/02/18'
    }
  }
  render() {
    return (
      <div>
      <h3>Edit Expense</h3>
        <form>
          <div>
            <label>Expense: </label><br />
            <input type="text"></input>
          </div>
          <div>
            <label>Cost: </label><br />
            <input type="text"></input>
          </div>
          <div>
            <label>Category: </label><br />
            <input type="text"></input>
          </div>
          <div>
            <label>Frequency: </label><br />
            <input type="text"></input>
          </div>
          <div>
            <label>Date: </label><br />
            <input type="text"></input>
          </div><br />
          <button type="submit">Save</button>
          <button type="submit">Cancel</button>
        </form>
      </div>
    )
  }
}

export default EditExpense;