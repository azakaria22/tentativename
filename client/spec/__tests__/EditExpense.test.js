import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

//Uncomment when using Redux and Passing state from store
//import configureStore from 'redux-mock-store'; //Smart components

import EditExpense from '../../src/components/EditExpense.jsx';

describe('<EditExpense />', () => {
  describe('render()', () => {
    test('renders the component', () => {
      const component = shallow(<EditExpense />);
      expect(toJson(component)).toMatchSnapshot();
    })
  })
})