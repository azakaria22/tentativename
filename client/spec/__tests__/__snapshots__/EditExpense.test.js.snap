// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<EditExpense /> render() renders the component 1`] = `
<div>
  <h3>
    Edit Expense
  </h3>
  <form>
    <div>
      <label>
        Expense: 
      </label>
      <br />
      <input
        type="text"
      />
    </div>
    <div>
      <label>
        Cost: 
      </label>
      <br />
      <input
        type="text"
      />
    </div>
    <div>
      <label>
        Category: 
      </label>
      <br />
      <input
        type="text"
      />
    </div>
    <div>
      <label>
        Frequency: 
      </label>
      <br />
      <input
        type="text"
      />
    </div>
    <div>
      <label>
        Date: 
      </label>
      <br />
      <input
        type="text"
      />
    </div>
    <br />
    <button
      type="submit"
    >
      Save
    </button>
    <button
      type="submit"
    >
      Cancel
    </button>
  </form>
</div>
`;
